enlighten/assets/example_data/mock_raman/mocks.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
I suggest that each record in this file get an optional "source_url" attribute that people can use to document where they got the peaks from.